csi/node.go

```diff
@@ -365,7 +365,8 @@ func (s *OsdCsiServer) NodeGetVolumeStats(ctx context.Context, req *csi.NodeGetV
 
 	var attachPathMatch bool
```

Comment:
In general, I don't understand the reason for this path match check. Regardless of the attachPath we can always return the volume usage for every volume. The attach path does not dictate the volume usage.